tests/autoskip/Earthfile

```diff
@@ -279,10 +279,15 @@ test-init-failure:
                                      --output_contains="Failed to initialize auto-skip database"
 
 test-arg-change:
+    # Please don't update. These versions are pinned to ensure a version
```

Comment:
I would reword this to be:

```
# this test requires two different versions of alpine
```

then add a following

```
RUN acbtest "$V1" != "$V2"
```

Reply:
the reasoning is to prevent cases where it's somehow auto-updated by an automated process, or user running a `sed` against all Earthfiles, which would miss such a comment.

Reply:
@alexcb Updated.